### DISCUSSION TITLE
Back button/gesture open up sidebar

### DISCUSSION THREAD

**Discussion by jpsnyder** [jocmp/capyreader#193] [Feature Requests]
When viewing the list of articles of a feed, I instinctively want to use the back gesture to get back to the sidebar to select another feed. But instead it takes my Unread (or exits if I'm already on Unread). I'm used to this from using the Infinity for Reddit app, and I'm sure there are some other apps that also do this. It would be nice if this were an option we can enable.

**Comment by jocmp** [comment]
hey @jpsnyder. I downloaded Infinity to check it out, but I couldn't find a setting or replicate this behavior. Is it a toggle or something?

**Reply by jpsnyder** [reply]
It's under miscellaneous under the settings. You can set the back button to open the navigation drawer.

![Screenshot_20240719-200232.png](https://github.com/user-attachments/assets/fed497a2-bc6c-4abb-a4e0-70129533d653)

**Reply by jocmp** [reply]
Nice, thanks. I'll remember this when I build out #196. The current settings page doesn't have a good spot for more customization.

**Reply by jocmp** [reply]
Swipe to mark read is available but overrides the drawer swipe. I want to add this as an option when that's added it to the Google Play build this week.

Infinity's options

<img src="https://github.com/user-attachments/assets/198523f0-cf37-4689-92a3-b524fae1f97c" width="300px">

**Comment by jpsnyder** [comment]
I like the new gesture to open the feeds list. I could probably train my muscle memory to do that instead. Eventually I'll stop accidentally closing the app :)

**Comment by jocmp** [comment]
Added as of 2024.09.1055
